tests/test_dataset_info.py

```diff
@@ -263,7 +263,7 @@ def test_pennsieve_identifier_dataset_search(client):
     assert 'result' in response
     assert len(response['result']) == 1
     result = response['result'][0]
-    assert "Human Islet Microvasculature Analysis" == result['name']
+    assert "Human islet microvasculature analysis" == result['name']
```

Comment:
Should we turn the strings to lower case for the dataset string test? It will break the test for production until the dev endpoints get promoted.

Reply:
Done 6343d14.